tests/unit/fake_data_root/openstack/sos_commands/apparmor/apparmor_status

```diff
@@ -255,6 +255,8 @@ apparmor module is loaded.
    snap.lxd.user-daemon
    virt-aa-helper
 0 profiles are in complain mode.
+0 profiles are in kill mode.
+0 profiles are in unconfined mode.
```

Comment:
Yeah I know it's a pain but if we start doing this it is a slippery slope. You could copy this file and append in your test maybe?